packages/terra-date-time-picker/src/DateTimeUtils.js

```diff
@@ -75,37 +75,52 @@ class DateTimeUtils {
    * Synchronize only the time for a given moment object.
    * @param {object} momentDate - The moment object to synchronize the date and time.
    * @param {string} time - The time to synchronize with the moment object.
+   * @param {boolean} seconds - If true seconds will be processed
    * @return {object} - The synchronized moment object.
    */
-  static updateTime(momentDate, time) {
+  static updateTime(momentDate, time, seconds) {
     if (!momentDate || !momentDate.isValid()) {
       return null;
     }
 
     const newDate = momentDate.clone();
-    const date = moment(time, 'HH:mm', true);
+    const date = moment(time, seconds ? 'HH:mm:ss' : 'HH:mm', true);
+
+    if (seconds) {
+      return newDate.hour(date.get('hour')).minute(date.get('minute')).second(date.get('second'));
+    }
 
     return newDate.hour(date.get('hour')).minute(date.get('minute'));
   }
 
+  /**
+   * Gets the time from a date and time
+   * @param {string} time An ISO 8601 string to get the time of
+   * @param {boolean} second Whether or not seconds are enabled
+   * @return {string} The time from the date and time string
+   */
+  static getTime(time, second) {
```

Comment:
I am not seeing any new jest tests to test this new function or to test the additional options of these existing helpers.